include/triton/core/tritonbackend.h

```diff
@@ -91,7 +91,7 @@ struct TRITONBACKEND_ModelInstance;
 ///   }
 ///
 #define TRITONBACKEND_API_VERSION_MAJOR 1
-#define TRITONBACKEND_API_VERSION_MINOR 9
+#define TRITONBACKEND_API_VERSION_MINOR 10
```

Comment:
Don't need to change version as there is no API changes

Reply:
The API hasn't changed but the functionality has. This lets the user know that they may see different behavior when using this version.